tests/helpers/test_config_entry_flow.py

```diff
@@ -33,17 +34,19 @@ async def has_discovered_devices(hass):
 
 
 @pytest.fixture
-def webhook_flow_conf(hass):
+def webhook_flow_conf(hass: HomeAssistant) -> Generator[None, None, None]:
     """Register a handler."""
     with patch.dict(config_entries.HANDLERS):
         config_entry_flow.register_webhook_flow("test_single", "Test Single", {}, False)
         config_entry_flow.register_webhook_flow(
             "test_multiple", "Test Multiple", {}, True
         )
-        yield {}
+        yield
```

Comment:
This is the only code change in the PR.
It returned an empty dict that was never read nor amended.
It was probably a copy-paste from the fixture above, and it seems more logical to simply return `None`.